smart_campus/app/forms.py

```diff
@@ -1,7 +1,7 @@
 from django import forms
```

Comment:
Is there any reason that you don't use `from django.forms import ModelForm`?

Reply:
None, I am curious as to it too, it was the last PR's commit

Reply:
okay, I will fixed it.